eventdata/parameter_sources/elasticlogs_bulk_source.py

```diff
@@ -86,7 +80,7 @@ def params(self):
         bulk_array = []
         for x in range(0, self._bulk_size):
             evt, idx, typ = self._randomevent.generate_event()
-            bulk_array.append('{"index": {"_index": "%s", "_type": "%s"}}"' % (idx, typ))
+            bulk_array.append('{"index": {"_index": "%s", "_type": "doc"}}"' % (idx))
```

Comment:
nit: if there is only one parameter in the format string, you can remove the parenthesis.